ehcache-impl/src/main/java/org/ehcache/impl/persistence/DefaultLocalPersistenceService.java

```diff
@@ -115,6 +140,17 @@ private void internalStart() {
   @Override
   public synchronized void stop() {
     if (started) {
+      try {
+        if (cleanFile.createNewFile()) {
+          LOGGER.debug("clean file is created.");
+        } else {
+          LOGGER.warn("clean file already exists. It's not deleted either user's permission or network issue." +
```

Comment:
This logging is verbose but doesn't really help the user much. What will happen as a result of this error/problem? What should the user do or expect?